values.yaml

```diff
@@ -32,6 +32,7 @@ service:
   # foo.io/bar: "true"
 ingress:
   enabled: false
+  ingressClassName: nginx
```

Comment:
`ingress.ingressClassName` is redundant -- instead, should be `ingress.className`.

Please add documentation for this value to the README as well.